CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - console: fix visiting view modify page overwriting raw sql content (fix #4798) (#4810)
 - console: add help button and move about page to settings (#4848)
 - console: add new sidebar icon that separates enums from tables (fix #4984) (#4992)
+- console: fix runtime error in `Browse Rows` page (fix #4907) (#5016)
```

Comment:
is it possible to explain the condition (as simply as possible) when this error is hit.

Reply:
its not reproduceable but the offending code is fixed. will update with the error type